docs/framework/additional-apis/_httpresponse.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "_HttpResponse Field | Microsoft Docs"
+ms.custom: ""
+ms.date: "04/24/2017"
+ms.prod: ".net-framework"
+ms.reviewer: ""
+ms.suite: ""
+ms.technology:
+  - "dotnet-winforms"
+ms.tgt_pltfrm: ""
+ms.topic: "article"
```

Comment:
the "template" topics got incorrectly migrated to articles when they were converted to MarkDown. the ms.topic value should be reference.